app/tour-step-directive.js

```diff
@@ -36,10 +36,12 @@ export function tourStepDirective(TourConfig, TourHelpers, uiTourService, $uibTo
                         }
                     },
                     events = 'onShow onShown onHide onHidden onNext onPrev onBackdropClick'.split(' '),
-                    options = 'content title animation placement backdrop backdropBorderRadius orphan popupDelay popupCloseDelay popupClass fixed preventScrolling scrollIntoView nextStep prevStep nextPath prevPath scrollOffset'.split(' '),
+                    options = 'content title animation placement backdrop backdropBorderRadius orphan popupDelay popupCloseDelay popupClass fixed preventScrolling scrollIntoView nextStep prevStep nextPath prevPath scrollOffset'.split(
+                        ' '),
```

Comment:
Can you bring this back up to the previous line so that `.split(' ' ),` is on the same line? Otherwise this looks good and I'll merge it.